osu.Game/Overlays/Settings/Sections/Maintenance/MigrationSelectScreen.cs

```diff
@@ -51,9 +52,9 @@ protected override void OnSelection(DirectoryInfo directory)
                     // Quick test for whether there's already an osu! install at the target path.
                     if (fileInfos.Any(f => f.Name == OsuGameBase.CLIENT_DATABASE_FILENAME))
                     {
-                        dialogOverlay.Push(new ConfirmDialog("The target directory already seems to have an osu! install. Use that data instead?", () =>
+                        dialogOverlay.Push(new ConfirmDialog(MaintenanceSettingsStrings.TargetDirectoryAlreadyInstalledOsu.ToString(), () =>
```

Comment:
Calling `ToString` destroys localisation support.

Reply:
Oh... I'll try to localize the ConfirmDialog side.